evap/contributor/views.py

```diff
@@ -84,7 +81,7 @@ def course_view(request, course_id):
         for field in cform.fields.values():
             field.disabled = True
 
-    template_data = dict(form=form, formset=formset, course=course, editable=False,
+    template_data = dict(form=form, formset=formset, course=course, editable=False, show_edit_message=False,
```

Comment:
...now after the simplification it seems that show_edit_message is now equivalent to `editable`, so as far as i can see you could just remove `show_edit_message` and use `editable`. could you verify that and implement it?